app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You not vaccinated!")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Your vaccine is lost date!")
```

Comment:
The error message 'Your vaccine is lost date!' is not clear. A more descriptive message would be 'Your vaccine is expired!' to clearly indicate the problem.